internal/auth/users.go

```diff
@@ -24,7 +25,9 @@ type User struct {
 	Name            string                    `json:"name" yaml:"name"`
 	Password        string                    `json:"-" yaml:"password"`
 	Filter          string                    `json:"-" yaml:"filter"`
+	Roles           string                    `json:"-" yaml:"roles"`
```

Comment:
Maybe this is just a personal taste but I would call this `RolesConfigred` and the other one just `Roles`. It feels better to write `user.roles.Has(...)` than `user.userRoles.HasRole(...)`. 

Not a big deal. I could always change it on my side.